packages/accordion/src/AccordionContainer.spec.tsx

```diff
@@ -9,35 +9,35 @@ import React from 'react';
 import { render, fireEvent } from '@testing-library/react';
 import { KEY_CODES } from '@zendeskgarden/container-utilities';
 import { AccordionContainer } from './AccordionContainer';
+import { IUseAccordionReturnValue } from './useAccordion';
 
 describe('AccordionContainer', () => {
   it('renders with expected return values', () => {
-    const { getByTestId } = render(
+    const { getByTestId, getByText } = render(
       <AccordionContainer>
-        {({ expandedSections, disabledSections }) => (
-          <div
-            data-test-id="test"
-            hidden={expandedSections.length === 0}
-            disabled={disabledSections.length !== 0}
-          />
+        {({ expandedSections, disabledSections }: IUseAccordionReturnValue) => (
+          <div data-test-id="test" hidden={expandedSections.length === 0}>
+            <button disabled={disabledSections.length !== 0}>Trigger</button>
+          </div>
         )}
       </AccordionContainer>
     );
     const test = getByTestId('test');
+    const triggerButton = getByText('Trigger');
 
     expect(test).toHaveAttribute('hidden');
-    expect(test).not.toHaveAttribute('disabled');
+    expect(triggerButton).not.toHaveAttribute('disabled');
   });
 
-  const sections = Array(3).fill();
+  const sections = Array(3).fill(undefined);
   const CONTAINER_ID_PREFIX = 'test';
 
   const BasicExample = ({
     expandedSections = [],
     expandable = true,
     collapsible = true,
     onChange
-  } = {}) => (
+  }: any = {}) => (
```

Comment:
This may not provide much help, but you could use `IUseAccordionProps` to type this correctly.

Reply:
That sounds great! I had initially tried that, but `IUseAccordionProps` does not include an `onChange` property, which is only used in the tests to pass a mock function. Is there a way I can use `IUseAccordionProps` without updating the interface to have an `onChange` property?

Reply:
You could try `{ onChange: (foo: Bar) => void; } & IUseAccordionProps`

Reply:
My mistake on this one, `onChange` **is** a property on `IUseAccordionProps`. This has been updated with your suggestion, thanks @austin94!